Doc/library/argparse-optparse.rst

```diff
@@ -0,0 +1,55 @@
+.. currentmodule:: argparse
+
+.. _upgrading-optparse-code:
```

Comment:
I feel this filename makes it look like it's docs for the `argparse.optparse` module. What do you think of moving it into `howto/` instead?

Reply:
Yeah - that's fair. Let me move that.